_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,407 @@
+---
+id: 9bd38c
+name: Non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when there is a visual reference of content, there are also non-visual indicators of the location.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: true
```

Comment:
@WilcoFiers pointed out that techniques are not required for conformance:
https://github.com/act-rules/act-rules.github.io/pull/1014#discussion_r344633847

Reply:
(and after talking with @kasperisager and checking the ACT rules format, I actually disagree… https://github.com/act-rules/act-rules.github.io/pull/1014#discussion_r346227590)

Reply:
We got to talk with @WilcoFiers and @kasperisager today and agreed that `forConformance: false` is the correct thing to have for techniques.

Reply:
Done! Thanks!